test/test-vault-policy-admin.hcl

```diff
@@ -0,0 +1,47 @@
+# Manage auth methods broadly across Vault
```

Comment:
I'm not reviewing these files, if they are incorrect it should be caught in testing and addressed in a followup PR